flink-connector-oracle-cdc/src/test/resources/docker/assets/activate-archivelog.sh

```diff
@@ -1,3 +1,4 @@
+#!/bin/bash
```

Comment:
Thanks @legendtkl for the contribution, but I checked the scripts works well even the `#!/bin/bash` is not placed on the first line. Could you explain more about the limitation?

Reply:
ref this: https://stackoverflow.com/questions/8967902/why-do-you-need-to-put-bin-bash-at-the-beginning-of-a-script-file

Reply:
Thanks @legendtkl for the clarification